packages/terra-notification-dialog/src/NotificationDialog.jsx

```diff
@@ -112,179 +103,119 @@ const propTypes = {
-                  {endMessage
-                    && <div className={cx('message')}>{endMessage}</div>}
+  /* eslint-disable jsx-a11y/no-noninteractive-tabindex */
+  return (
+    <AbstractModal
+      ariaLabel="Notification Dialog"
+      aria-labelledby="notification-dialog-header"
+      aria-describedby={title ? 'notification-dialog-title' : 'notification-dialog-header'}
+      role="alertdialog"
+      classNameModal={classNames(cx('notification-dialog', theme.className), customProps.className)}
+      isOpen={isOpen}
+      onRequestClose={() => {}}
+      closeOnEsc={false}
+      closeOnOutsideClick={false}
+      zIndex="9000"
+      data-terra-notification-dialog
+    >
+      <FocusTrap focusTrapOptions={{ returnFocusOnDeactivate: true, clickOutsideDeactivates: false, escapeDeactivates: false }}>
+        <div className={cx('notification-dialog-inner-wrapper')}>
+          <div className={cx('notification-dialog-container')} tabIndex="0">
+            <div className={cx(['floating-header-background', variant])} />
+            <div className={cx(['header'])}>
+              <div className={cx(['header-content'])}>
+                <NotificationIcon variant={variant} iconClassName={(custom || {}).iconClassName} />
+                <div className={cx('header-container')}>
+                  <div id="notification-dialog-signal-word" className={cx('signal-word')}>{signalWord}</div>
+                  <div id="notification-dialog-title" className={cx('title')}>{title}</div>
                 </div>
               </div>
-              <div className={cx('footer-body')}>
-                {actionSection(
-                  acceptAction || primaryAction,
-                  rejectAction || secondaryAction,
-                  buttonOrder,
-                  emphasizedAction,
-                )}
-              </div>
+            </div>
+            <div className={cx('body')}>
+              {(startMessage)
+                && <div className={cx('message')}>{(startMessage)}</div>}
+              {content
+                && <div className={cx('message')}>{content}</div>}
+              {endMessage
+                && <div className={cx('message')}>{endMessage}</div>}
+            </div>
+            <div className={cx('footer')}>
+              {actionSection(
+                acceptAction,
+                rejectAction,
+                buttonOrder,
+                emphasizedAction,
+              )}
             </div>
           </div>
-        </FocusTrap>
-      </AbstractModal>
-    );
-    /* eslint-enable jsx-a11y/no-noninteractive-tabindex */
-  }
-}
+        </div>
+      </FocusTrap>
+    </AbstractModal>
+  );
+  /* eslint-enable jsx-a11y/no-noninteractive-tabindex */
+};
 
 NotificationDialog.propTypes = propTypes;
 NotificationDialog.defaultProps = defaultProps;
 
-export { variants as NotificationDialogVariants };
+export { variants as NotificationDialogVariants, ContentLayoutAsList };
```

Comment:
I think we've discussed before discontinuing the practice of exporting variants like this. In practice it's really much easier to just type the string, would we want to possibly remove this with the MVB?

Reply:
removed: https://github.com/cerner/terra-framework/pull/1105/commits/022584400b4aaafcf4ad53ff1ed2e2b2eaec08a5#